src/types/chat.ts

```diff
@@ -17,6 +18,7 @@ export default interface Chat {
   readonly msg: string;
   readonly room: Room;
   readonly user: User;
+  readonly links: Link[];
```

Comment:
@arminfro after the pull I've got this line twice. I'll remove the second one on my branch.